Client/Frontend/Theme/LegacyThemeManager/LegacyThemeManager.swift

```diff
@@ -96,8 +96,11 @@ class LegacyThemeManager {
     private func applicationDidBecomeActive() {
         let nightMode = UserDefaults.standard.bool(forKey: "profile.NightModeStatus")
         if !nightMode && LegacyThemeManager.instance.systemThemeIsOn {
-            let userInterfaceStyle = UIScreen.main.traitCollection.userInterfaceStyle
+            /* Ecosia: Update legacy theme accoridng correctly
+            let userInterfaceStyle = traitCollection.userInterfaceStyle
```

Comment:
I believe the old commented code used `UIScreen` already so this comment here is wrong

```suggestion
            let userInterfaceStyle = UIScreen.main.traitCollection.userInterfaceStyle
```